### PR TITLE
LRAC-5856 Verify pageUnloaded event functionality

### PR DESCRIPTION
**Pull request by fcodias**
https://issues.liferay.com/browse/LRAC-5856

cc @marcellustavares

### CODE REVIEW

**Comment by fcodias** [comment]
ci:test:sf

**Comment by andreldm** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/analytics/analytics-client-js/src/plugins/dxp.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ * Plugin function that registers listeners related to DXP
+ * @param {Object} analytics The Analytics client
+ */
+
+function dxp(analytics) {
+
+	/**
+	 * Sends page load information on the endNavigate event when SPA is enabled on DXP
+	 */
+	function endNavigate() {
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/plugins/dxp.js:54]: Sort?

**Comment by fcodias** [comment]
ci:test:sf

**Comment by andreldm** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by andreldm** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by andreldm** [comment]
ci:forward

**Pull request closed without merging**